languages/Language.php

```diff
@@ -4090,4 +4090,32 @@ public function getPluralRules() {
 		return $pluralRules;
 	}
 
+	/**
+	 * Shorten number to thousands/millions/billions
+	 *
+	 * @param $number
+	 * @return String
+	 *
+	 * (added by Wikia)
+	 */
+	public function shortenNumberDecorator( $number ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)